src/timeMachine/components/view_options/TimeAutoToggle.tsx

```diff
@@ -0,0 +1,38 @@
+import React, {FC, useState} from 'react'
+import {useSelector} from 'react-redux'
+import {SlideToggle, ComponentSize} from '@influxdata/clockface'
+
+import {getXColumnTimeDomainBounds} from 'src/timeMachine/selectors'
+
+interface Props {
+  onSetDomain: (xDomain: [number, number]) => void
+}
+
+const TimeDomainAutoToggle: FC<Props> = ({onSetDomain}) => {
```

Comment:
```suggestion
export const TimeDomainAutoToggle: FC<Props> = ({onSetDomain}) => {
```

default exports aren't great in a team environment. They make searching a large codebase like ours difficult, they lead to inconsistent naming on a 12 person team like ours, and they make tree shaking and code splitting less effective.

https://basarat.gitbook.io/typescript/main-1/defaultisbad